### ISSUE TITLE
Missing dependencies

### ISSUE THREAD

**Issue by ericovis** [discovery/django-bitly#20]
Hey, I'm new to this but it appears that you have some missing dependencies:

```
System check identified no issues (0 silenced).
Unhandled exception in thread started by <function wrapper at 0x10a7a3c08>
Traceback (most recent call last):
  File "/Users/ericovis/Projects/emagalhaes/env/lib/python2.7/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/Users/ericovis/Projects/emagalhaes/env/lib/python2.7/site-packages/mezzanine/core/management/commands/runserver.py", line 163, in inner_run
    super(Command, self).inner_run(*args, **kwargs)
  File "/Users/ericovis/Projects/emagalhaes/env/lib/python2.7/site-packages/django/core/management/commands/runserver.py", line 124, in inner_run
    self.check_migrations()
  File "/Users/ericovis/Projects/emagalhaes/env/lib/python2.7/site-packages/django/core/management/base.py", line 437, in check_migrations
    executor = MigrationExecutor(connections[DEFAULT_DB_ALIAS])
  File "/Users/ericovis/Projects/emagalhaes/env/lib/python2.7/site-packages/django/db/migrations/executor.py", line 20, in __init__
    self.loader = MigrationLoader(self.connection)
  File "/Users/ericovis/Projects/emagalhaes/env/lib/python2.7/site-packages/django/db/migrations/loader.py", line 52, in __init__
    self.build_graph()
  File "/Users/ericovis/Projects/emagalhaes/env/lib/python2.7/site-packages/django/db/migrations/loader.py", line 197, in build_graph
    self.load_disk()
  File "/Users/ericovis/Projects/emagalhaes/env/lib/python2.7/site-packages/django/db/migrations/loader.py", line 108, in load_disk
    migration_module = import_module("%s.%s" % (module_name, migration_name))
  File "/usr/local/Cellar/python/2.7.12_2/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/Users/ericovis/Projects/emagalhaes/env/lib/python2.7/site-packages/django_bitly/migrations/0002_auto__chg_field_stringholder_absolute_url.py", line 2, in <module>
    from south.utils import datetime_utils as datetime
ImportError: No module named south.utils
```

**Comment by fcurella** [comment]
Hi @ericovis , thanks for the report!

Yeah, migrations needs to be converted from south to django. I'm not sure I'll have time today, I'll try to get something later this week

**Comment by fcurella** [comment]
I've converted the migrations to using Django's native system. Since this is a breaking change, I've released a new major version, `v1.0.1`.
